### PR TITLE
Role Model

### PR DESCRIPTION
**Pull request by YumilRuedaFlores-Ksquare**

### CODE REVIEW

**Review by ArgenisGonzalez-Ksquare — DISMISSED** [review]

**Review by MarianaLopez-Ksquare — APPROVED** [review]

File: src/models/Role.model.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Model, InferAttributes, InferCreationAttributes, CreationOptional, DataTypes, Sequelize } from "sequelize";
+
+
+export class Role extends Model<InferAttributes<Role>, InferCreationAttributes<Role>> {
+
+    declare id: number;
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Role.model.ts:6]: Maybe CreationOptional?

Comment (YumilRuedaFlores-Ksquare) [reply src/models/Role.model.ts]: Changed

**Review by ArgenisGonzalez-Ksquare — DISMISSED** [review]

File: src/models/Role.model.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    
+        id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        },
+        role: DataTypes.STRING
+    
+}, {
+    tableName: "Role"
```
Comment (GabrielOsorno-Ksquare) [inline src/models/Role.model.ts:22]: Would be great if you change it to 'role' c:, just for consistency

File: src/models/Role.model.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    Role.init({
+    
+        id: {
+            type: DataTypes.INTEGER,
+            autoIncrement: true,
+            primaryKey: true
+        },
+        role: {
+            type: DataTypes.STRING,
+            allowNull: false
```
Comment (YumilRuedaFlores-Ksquare) [inline src/models/Role.model.ts:21]: Allownull

**Review by GabrielOsorno-Ksquare — APPROVED** [review]

**Review by ClaudiaDiaz-Ksquare — APPROVED** [review]

**Review by ArgenisGonzalez-Ksquare — APPROVED** [review]

**Review by IvannaNouel-Ksquare — APPROVED** [review]

**Pull request merged**